platform/darwin/include/mbgl/storage/reachability.h

```diff
@@ -69,8 +69,6 @@ typedef void (^NetworkUnreachable)(MGLReachability * reachability);
 +(instancetype)reachabilityWithAddress:(void *)hostAddress;
 +(instancetype)reachabilityForLocalWiFi;
 
--(instancetype)initWithReachabilityRef:(SCNetworkReachabilityRef)ref;
```

Comment:
Out of curiosity, what static analyzer warning did removing this method resolve?

Reply:
It didn't 😄 

I removed it from the header, because I had previously added `CFRetain`ing of `ref` that is passed in i.e. a change in behavior. 

Fortunately there were no other consumers of this method.